CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,77 @@
+# Changelog 1.15
+
+## 1.15.0 - 2024-01-26
+
+### Added
+
+- Add a new feature `NodeNetworkPolicy` to allow user to apply `ClusterNetworkPolicy` to Kubernetes Nodes. ([#5658](https://github.com/antrea-io/antrea/pull/5658) [#5716](https://github.com/antrea-io/antrea/pull/5716), [@hongliangl] [@Atish-iaf])
+- Add L7 network flow export support in Antrea to enables users to export L7 protocol information. ([#5218](https://github.com/antrea-io/antrea/pull/5218), [@tushartathgur])
+- Add migration tool to support migrating from other CNIs to Antrea. ([#5677](https://github.com/antrea-io/antrea/pull/5677), [@hjiajing])
+- Support Egress using IPs from a separate subnet through VLAN. ([#5799](https://github.com/antrea-io/antrea/pull/5799), [@tnqn])
+- Record an event when Egress IP assignment changes for better troubleshooting. ([#5765](https://github.com/antrea-io/antrea/pull/5765), [@jainpulkit22])
+- Add Support for `FlexibleIPAM` on Antrea Multicast so that multicast packets can be forwarded to the uplink interface when `FlexibleIPAM` is enabled. ([#4922](https://github.com/antrea-io/antrea/pull/4922), [@ceclinux])
+- Support Talos clusters to run Antrea as the CNI, and add Talos to the K8s installers document. ([#5718](https://github.com/antrea-io/antrea/pull/5718) [#5766](https://github.com/antrea-io/antrea/pull/5766), [@antoninbas])
+
+### Changed
+
+- Support secondary network when the network configuration in `NetworkAttachmentDefinition` does not include IPAM configuration. ([#5762](https://github.com/antrea-io/antrea/pull/5762), [@jianjuns])
+- Change secondary network Pod controller to subscribe to CNIServer events to support bridging and VLAN network. ([#5767](https://github.com/antrea-io/antrea/pull/5767), [@jianjuns])
+- Use Antrea IPAM for Antrea native secondary network support. ([#5427](https://github.com/antrea-io/antrea/pull/5427), [@jianjuns])
+- Build different images for antrea-agent and antrea-controller to reduce the image size, which can accelerate the antrea-agent and antrea-controller startup. ([#5856](https://github.com/antrea-io/antrea/pull/5856) [#5902](https://github.com/antrea-io/antrea/pull/5902) [#5903](https://github.com/antrea-io/antrea/pull/5903), [@jainpulkit22])
```

Comment:
Might prefer like "Create different images for antrea-agent and antrea-controller to minimize the overall image size, speeding up the startup of both antrea-agent and antrea-controller"

Reply:
Refined.